js/interface-lists.js

```diff
@@ -510,9 +510,10 @@ function attahObservers() {
   });
 
   Fliplet.Widget.onSaveRequest(function () {
-    if (dynamicLists.isLoaded) {
-      $('form').submit();
+    if (!dynamicLists.isLoaded) {
+      Fliplet.Widget.complete();
```

Comment:
@maksim-an-upplabs Can you explain why you haven't stopped the code execution with `return` after this? Otherwise, it looks like if the configuration isn't loaded, it will still run `$('form').submit()`, which would replicate the reported bug.